src/Stratis.Bitcoin.Tests/NodeConfiguration/NodeSettingsTest.cs

```diff
@@ -19,9 +20,9 @@ public NodeSettingsTest():base(Network.Main)
         public void NodeSettings_CanReadSingleValueFromCmdLine()
         {
             // Arrange
-            var nodeSettings = new NodeSettings(args:new[] { "-agentprefix=abc" });
+            var connectionSettings = new ConnectionManagerSettings(new NodeSettings(args:new[] { "-agentprefix=abc" }));
```

Comment:
I'd rather you changed these tests to test properties in NodeSettings rather than now moving to test the properties in ConcectionManagerSettings.

Reply:
Changed.